packages/math-input/src/components/input/math-input.tsx

```diff
@@ -263,13 +228,10 @@ class MathInput extends React.Component<Props, State> {
         this._root.style.fontSize = `${fontSizePt}pt`;
     };
 
-    /** Gets and cache they bounds of the keypadElement */
+    /** Returns the current bounds of the keypadElement */
     _getKeypadBounds: () => any = () => {
-        if (!this._keypadBounds) {
-            const node = this.props.keypadElement?.getDOMNode();
-            this._cacheKeypadBounds(node);
-        }
-        return this._keypadBounds;
+        const keypadNode = this.props.keypadElement?.getDOMNode() as Element;
```

Comment:
I know this would be harder to type, but I think this would be safer.

```suggestion
        const keypadNode = this.props.keypadElement?.getDOMNode();
        
        if (!keypadNode) {
            return null
        }
```

Reply:
This ended up working nicely with my new "isWithinKeypadBounds" function from the other PR